package.json

```
@@ -1,11 +1,12 @@
 {
   "name": "joi-extract-type",
-  "version": "15.0.8",
+  "version": "16.0.0",
```

Lets' just be in sync with the Joi version, if that's ok.
The idea with this was to make it easier to understand the support.

```suggestion
  "version": "17.0.0",
```